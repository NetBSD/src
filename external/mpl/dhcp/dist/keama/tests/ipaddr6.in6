# IPv6 addresses config

option dhcp6.name-servers 2001::, 200a::0bF, 2001::192.168.0.1;
