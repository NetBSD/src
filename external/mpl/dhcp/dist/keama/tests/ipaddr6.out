{
  # IPv6 addresses config
  "Dhcp6": {
    "option-data": [
      {
        "space": "dhcp6",
        "name": "dns-servers",
        "code": 23,
//      "original-data": "2001::, 200a::0bF, 2001::192.168.0.1",
        "data": "2001::, 200a::bf, 2001::c0a8:1"
      }
    ]
  }
}
